styles/layout.css

```diff
@@ -1,14 +1,27 @@
-.template {
-  display: none;
+#articles {
   text-align: center;
-  margin: 0 auto;
 }
 
-a {
```

Comment:
I would look into using classes to style and not ids. I don't know why exactly but i think it's good practice.